### ISSUE TITLE
Changes in the Gut Microbiome and Predicted Functional Metabolic Effects in an Australian Parkinson's Disease Cohort

### ISSUE THREAD

**Issue by Omabekee** [waldronlab/BugSigDBcuration#266]
Changes in the Gut Microbiome and Predicted Functional Metabolic Effects in an Australian Parkinson's Disease Cohort -- Kenna et al. -- Frontiers in Neuroscience
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8588830/

**Comment by AdegboyeRukayat** [comment]
@SvetlanaUP Kindly assign me this paper to curate

**Comment by SvetlanaUP** [comment]
@AdegboyeRukayat is your previous curation ready for review https://github.com/waldronlab/BugSigDBcuration/issues/205 ?

**Comment by AdegboyeRukayat** [comment]
@SvetlanaUP Yes please

**Comment by SvetlanaUP** [comment]
> @SvetlanaUP Yes please

Could you please share a link to your BugSigDB curation? Thanks @AdegboyeRukayat

**Comment by Nikilight** [comment]
Hello @SvetlanaUP and @Omabekee I have put up my second contribution for review. While I await your review, may I be assigned this Article for another contribution?
I look forward to your response.
Thank you.

**Comment by SvetlanaUP** [comment]
Hi @Nikilight I'm waiting for @AdegboyeRukayat response, only then I will know if this paper is available or not. Thanks for your patience!

**Comment by Nikilight** [comment]
Alright. That is noted @SvetlanaUP . Thank you for your response.

**Comment by SvetlanaUP** [comment]
Assigned to @Nikilight

**Comment by Nikilight** [comment]
Thank you @SvetlanaUP

**Comment by AdegboyeRukayat** [comment]
@SvetlanaUP kindly help me review my second contribution https://bugsigdb.org/Study_925

**Comment by nityasinghal-14** [comment]
Hey @Nikilight  , do you need any help? I am available and would love to collaborate with you.

**Comment by Nikilight** [comment]
Hello @nityasinghal-14 This is so thoughtful of you and I appreciate you. I was checking through my microbial signatures before now. I'd edit it soon and put it up for review. 
Once again, thank you.

**Comment by Nikilight** [comment]
Good day @SvetlanaUP , @Omabekee. My curation is ready for your reviews. 
https://bugsigdb.org/Study_973 
Thank you.

**Comment by Scholarpat** [comment]
Hi @SvetlanaUP,

Here are the changes I made to this curation:

- Statistical Test: Changed from 'ANOVA' to 'Wilcoxon'.
- Matched-on: Controls were not matched on any factor, so I removed 'age' which was initially curated.
- Signature: Some taxa were not curated, including Proteobacteria, Synergistetes, and Enterobacterales for Signature 1, and Clostridia and Clostridiales for Signature 2. I've added them.
- Clostridiales bacterium was initially curated in place of Clostridiales, which was reported. I've replaced this with its heterotypic synonym 'Eubacteriales'.

The curation is complete.

Thank you!
